drivers/node/ssh/ssh.go

```diff
@@ -325,6 +325,14 @@ func (s *SSH) RunCommand(n node.Node, command string, options node.ConnectionOpt
 	return output.(string), nil
 }
 
+// RunCommandWithoutRetry runs given command on given node but with no retries
+func (s *SSH) RunCommandWithoutRetry(n node.Node, command string, options node.ConnectionOpts) (string, error) {
```

Comment:
Is there a way we can refactor `RunCommand` to call `RunCommandWithoutRetry`, or have them share a common function somehow? My main concern is that I see `RunCommand` has some special error returns (that `&node.ErrFailedToRunCommand`), and while we could just fix that specific case to make this function also return that, it would be better if we could have them share the same logic so we only have to change it in one place in the future.